### ISSUE TITLE
Implement revised CINERGI detail view

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/model-my-watershed#2393]
Currently looks like:
![screen shot 2017-10-19 at 10 58 45 am](https://user-images.githubusercontent.com/12868342/31777630-800e20ba-b4bc-11e7-8071-e2cf49b4478c.png)

Waiting on mockup from Anthony + Emilio.

**Comment by aufdenkampe** [comment]
@emiliom, can you update us on progress on the Details page for a dataset from CINERGI?

**Comment by emiliom** [comment]
Sorry, I was hoping to be done by now. I'm working on it now and will plan to be done by 2:30pm PT. Will let you know here.

**Comment by emiliom** [comment]
Ok, my best shot is ready: https://docs.google.com/document/d/1k9_I1gp9V8F2ieL-LfN2OYCimrrqyf9IKAGtCSlM8Uw/edit?usp=sharing

@aufdenkampe, please take a look if you have a chance. It's not straightforward what should be kept! Also, I don't have a url for the CINERGI API itself (I tried looking for it), so I've only provided a link to the CINERGI catalog proper. If you have it, please replace it and copy the CINERGI catalog link to an "info" link in the Catalog row (just like HydroShare, I think)

**Comment by emiliom** [comment]
One possible glitch: according to [this](1929#issue-232377915), `links_s` "field may be a string or list of strings"; if it is a list of strings, use the first one only.

**Comment by arottersman** [comment]
@aufdenkampe, @emiliom, thanks for providing that template. Should there be an info tooltip next to "Catalog: CINERGI" like we've done for HydroShare, and if so, what should it contain?

**Comment by aufdenkampe** [comment]
Tooltip with this text and link:
"The Community Inventory of EarthCube Resources for Geosciences Interoperability (CINERGI) Data Portal is a catalog of databases, services and data portals developed by numerous organizations representing a diversity of geoscience domains. Explore CINERGI at http://cinergi.sdsc.edu/."

**Comment by emiliom** [comment]
Thanks, Anthony. That looks good.

**Comment by emiliom** [comment]
Thanks, Anthony. That looks good.

**Comment by rajadain** [comment]
Hello, some follow up questions regarding the organizations listed in details view:

 1. In case of large number of organizations, should we show all of them or only the top `n`?

    ![jauqolzvnr](https://user-images.githubusercontent.com/7633670/32016241-62c6ce4c-b991-11e7-8431-7bf8cf691c18.gif)

 2. Should we de-duplicate repeating organizations?

    <img width="460" alt="screen shot 2017-10-25 at 1 58 15 pm" src="https://user-images.githubusercontent.com/7633670/32016312-979296e2-b991-11e7-9db2-a0547a4b1cee.png">

 3. In case of missing contacts, should we still show the "Contacts" header?

    ![image](https://user-images.githubusercontent.com/1430060/32063964-917a7c60-ba46-11e7-885b-0cbb33ca225f.png)

**Comment by aufdenkampe** [comment]
Earlier, I had proposed to Emilio that we might want to combine the contact_people and contact_organization lists (similar to how we did it for the HydroShare detail page) like this:

`Contacts:	<_source/contact_people_s[0]>, <_source/contact_organizations_s[0]>
		<_source/contact_people_s[1]>, <_source/contact_organizations_s[1]>
		...
`
@emiliom pointed out that, unfortunately, contact_people_s[0]> isn't always affiliated with <_source/contact_organizations_s[0]>, etc. In other words, sometimes the proper order of the two lists is not maintained.

It seems to me that we might be able to eliminate all three of @rajadain's issues by trying out the combined version (and replacing the word "missing" with a null value).

I realize that we could get things wrong sometimes, but perhaps that would motivate corrections in the original dataset record.

@rajadain, could you try the combined approach above?

**Comment by emiliom** [comment]
Note that the mismatch is not just about order. There isn't always a 1:1
correspondence between the contact and organization lists.

Either way, I definitely recommend de-duplication for contacts and orgs
(regardless of whether they're combined or remain separated)

**Comment by aufdenkampe** [comment]
It would make sense that sometimes a organization might be listed without a person contact.

@rajadain could test if the list lengths are the same before combining. If the Organization list is longer, I'm guessing that the first records won't have a person associated with them. @valentinedwv (from CINERGI), could you let us know if there was any convention to how contact_people_s> match with <_source/contact_organizations_s> when the lists are not of the same length?

@rajadain, I'll let you decide which of the two implementations makes the most sense, given our time constraints. Getting this perfect is of lower priority than moving forward with WDC free text search.

If the lists are kept separate, I recommend:
1. Keeping all Organizations, with will get shorter by...
2. de-duplicating repeating organizations
3. Leaving "Contacts",  when there are no people, and adding "none provided" or some other text to show that the record does not contain that information.

**Comment by rajadain** [comment]
Did the above, it looks better now:

![image](https://user-images.githubusercontent.com/1430060/32067504-7ed3a86a-ba51-11e7-8b52-1e75ac332813.png)
